src/main/ErrorMessages.js

```diff
@@ -38,6 +38,14 @@ const SESSION_PROCESS_ERROR_MSGS = {
     SESSION_NOT_IN_USE: 'Session is not in use',
 }
 
+const FUNCTION_PARAMETERS_ERROR_MSGS = {
+    MISSING_KEY: (key,callingFunctionName) => `Missing key: ${key}. ${callingFunctionName}`,
```

Comment:
```suggestion
    MISSING_KEY: (key,callingFunctionName) => `Missing key: '${key}' when using '${callingFunctionName}'`,
```
Fix other messages in the same way

Reply:
callingFunctionName do it automatically, changing for the key